lib/modules/datasource/bitbucket-server-tags/index.ts

```diff
@@ -0,0 +1,194 @@
+import { ZodError } from 'zod';
+import { logger } from '../../../logger';
+import { cache } from '../../../util/cache/package/decorator';
+import type { PackageCacheNamespace } from '../../../util/cache/package/types';
+import { BitbucketServerHttp } from '../../../util/http/bitbucket-server';
+import { regEx } from '../../../util/regex';
+import { Result } from '../../../util/result';
+import { ensureTrailingSlash } from '../../../util/url';
+import { Datasource } from '../datasource';
+import { DigestsConfig, ReleasesConfig } from '../schema';
+import type { DigestConfig, GetReleasesConfig, ReleaseResult } from '../types';
+import {
+  BitbucketServerCommits,
+  BitbucketServerTag,
+  BitbucketServerTags,
+} from './schema';
+
+export class BitbucketServerTagsDatasource extends Datasource {
+  static readonly id = 'bitbucket-server-tags';
+
+  override http = new BitbucketServerHttp(BitbucketServerTagsDatasource.id);
+
+  static readonly defaultRegistryUrls = ['https://bitbucket.org'];
```

Comment:
```suggestion
  static readonly defaultRegistryUrls = [];
```
this would crash and never work. `bitbucket.org` has a totally different platform / api

Reply:
I've removed this fallback now and added an explicit check for `registryUrl`. It wouldn't have crashed but arguably not have been very useful either, since requests would have always errored.